clarity/src/vm/docs/mod.rs

```diff
@@ -1889,79 +1886,79 @@ mod test {
         }
     }
 
-    fn docs_execute(marf: &mut MarfedKV, program: &str) {
-        // start the next block,
-        //  we never commit it so that we can reuse the initialization
-        let mut store = marf.begin(&StacksBlockId([0; 32]), &StacksBlockId([1; 32]));
-
-        // execute the program, iterating at each ";; Returns" comment
-        // there are maybe more rust-y ways of doing this, but this is the simplest.
-        let mut segments = vec![];
-        let mut current_segment: String = "".into();
-        for line in program.lines() {
-            current_segment.push_str(line);
-            current_segment.push_str("\n");
-            if line.contains(";;") && line.contains("Returns ") {
-                segments.push(current_segment);
-                current_segment = "".into();
-            }
-        }
-        if current_segment.len() > 0 {
-            segments.push(current_segment);
-        }
-
-        let contract_id = QualifiedContractIdentifier::local("docs-test").unwrap();
-
-        {
-            let mut analysis_db = store.as_analysis_db();
-            let whole_contract = segments.join("\n");
-            eprintln!("{}", whole_contract);
-            let mut parsed = ast::build_ast(&contract_id, &whole_contract, &mut ())
-                .unwrap()
-                .expressions;
-
-            type_check(&contract_id, &mut parsed, &mut analysis_db, false)
-                .expect("Failed to type check");
-        }
-
-        let conn = store.as_clarity_db(&DOC_HEADER_DB, &DOC_POX_STATE_DB);
-        let mut contract_context = ContractContext::new(contract_id.clone());
-        let mut global_context = GlobalContext::new(
-            false,
-            conn,
-            LimitedCostTracker::new_free(),
-            StacksEpochId::Epoch2_05,
-        );
-
-        global_context
-            .execute(|g| {
-                for segment in segments.iter() {
-                    let expected = if segment.contains("Returns ") {
-                        let expects_start = segment.rfind("Returns ").unwrap() + "Returns ".len();
-                        Some(segment[expects_start..].trim().to_string())
-                    } else {
-                        None
-                    };
-
-                    eprintln!("{}", segment);
-
-                    let result = {
-                        let parsed = ast::build_ast(&contract_id, segment, &mut ())
-                            .unwrap()
-                            .expressions;
-                        eval_all(&parsed, &mut contract_context, g).unwrap()
-                    };
-
-                    if let Some(expected) = expected {
-                        assert_eq!(expected, result.unwrap().to_string());
-                    }
-                }
-                Ok(())
-            })
-            .unwrap();
-
-        store.rollback_block();
-    }
+    // fn docs_execute(marf: &mut MarfedKV, program: &str) {
```

Comment:
Is this going away?

Reply:
Nope -- docs tests and generation reenabled in 84b4369485e13bc1127bb2eb65fbd053343917d2